core/server/themes/index.js

```diff
@@ -45,5 +52,16 @@ module.exports = {
     loadOne: themeLoader.loadOneTheme,
     list: require('./list'),
     validate: validate,
-    toJSON: require('./to-json')
+    toJSON: require('./to-json'),
+    getActive: active.get,
+    activate: function activate(loadedTheme, checkedTheme) {
+        if (!_.has(checkedTheme, 'results.score.level') || checkedTheme.results.score.level !== 'passing') {
```

Comment:
If i am getting this right, we can't activate a theme which is/was scored with `warning`?
Because the error case is already handled here, https://github.com/TryGhost/Ghost/blob/master/core/server/themes/validate.js#L21?
Tell me if i am reading this wrong.